query-planner-js/src/QueryPlan.ts

```diff
@@ -90,7 +90,7 @@ export interface FetchDataKeyRenamer {
 export interface FlattenNode {
   kind: 'Flatten';
   path: ResponsePath;
-  node: PlanNode;
+  node?: PlanNode;
```

Comment:
this change was made just because there can be an undefined node returned from FetchGroup.toPlanNode()

Reply:
Figure out a way to handle the undefined node here.